lib/fathead/pep/output.txt

```diff
@@ -0,0 +1,421 @@
+Backport ensurepip (PEP 453) to Python 2.7	A			477							<section class="prog__container"><div class="section" id="abstract">  <p>     This PEP proposes that the     <code class="inline">      ensurepip     </code>     module, added to Python 3.4 by           PEP 453          , be backported to Python 2.7. It also proposes that automatic invocation of     <code class="inline">      ensurepip     </code>     be added to the Python 2.7 Windows and OSX installers. However it does     <strong>      not     </strong>     propose that automatic invocation be added to the     <code class="inline">      Makefile     </code>     .    </p> <p>     It also proposes that the documentation changes for the package distribution and installation guides be updated to match that in 3.4, which references using the     <code class="inline">      ensurepip     </code>     module to bootstrap the installer.    </p> </div></section>	https://www.python.org/dev/peps/pep-0477/
+PEP 1 - PEP Purpose and Guidelines	A			1							<section class="prog__container"><div class="section" id="what-is-a-pep">  <p>     PEP stands for Python Enhancement Proposal.  A PEP is a design document providing information to the Python community, or describing a new feature for Python or its processes or environment.  The PEP should provide a concise technical specification of the feature and a rationale for the feature.    </p> <p>     We intend PEPs to be the primary mechanisms for proposing major new features, for collecting community input on an issue, and for documenting the design decisions that have gone into Python.  The PEP author is responsible for building consensus within the community and documenting dissenting opinions.    </p> <p>     Because the PEPs are maintained as text files in a versioned repository, their revision history is the historical record of the feature proposal                     .    </p> </div></section>	https://www.python.org/dev/peps/pep-0001/
```

Comment:
@pjhampton we use the article title as the search term we trigger the Fathead on. By modifying the titles, it requires the user to search this specific string to see the answer.

I can think of two solutions:

1) We create redirects for `PEP #` to `PEP # - {title}`
2) We put the written title as part of the content:

```html
<section class="prog__container">
  <span class="prog__sub">Title</span>
  <p>{{title}}</p>
  <span class="prog__sub">Description</span>
  <p>{{description}}</p>
</section>
```